lib/info.js

```diff
@@ -443,6 +443,17 @@ for (let funcName of ['getBasicInfo', 'getInfo']) {
   };
 }
 
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
+miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
```

Comment:
put inside the function above, at line 440. otherwise, this is called only once when ytdl-core is first required. i also think this would all look cleaner if it was encapsulated in a function called `checkForUpdate` and then calling that function.